twistlock/README.md

```diff
@@ -0,0 +1,48 @@
+# Agent Check: Twistlock
+
+## Overview
+
+This check monitors [Twistlock][1].
+
+## Setup
+
+### Installation
+
+The Twistlock check is included in the [Datadog Agent][2] package, so you do not
+need to install anything else on your server.
+
+### Configuration
+
+1. Edit the `twistlock.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+   Agent's configuration directory to start collecting your twistlock performance data.
+   See the [sample twistlock.d/conf.yaml][2] for all available configuration options.
+
+2. [Restart the Agent][3]
+
+### Validation
+
+[Run the Agent's `status` subcommand][4] and look for `twistlock` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+Twistlock does not include any metrics.
```

Comment:
If it's not collecting any metrics/service Checks/ events.. what does this integration do? 🤔 🤔

Reply:
@l0k0ms it has the WIP in it, this is unfinished and I haven't even filled that out

Reply:
It looks like this whole implementation is gonna be thrown out anyway